src/screens/PostPageDetail/index.js

```diff
@@ -51,4 +50,4 @@ const styles = StyleSheet.create({
   }
 });
 
-export default withInteractionsManaged(FeedsPostDetail);
+export default FeedsPostDetail;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `FeedsPostDetail` component is now being exported directly instead of being wrapped with `withInteractionsManaged`. If `withInteractionsManaged` was used to improve performance by managing interactions, removing it might lead to performance issues. Ensure that the removal of this wrapper does not degrade the performance or user experience.
```diff
- export default withInteractionsManaged(FeedsPostDetail);
+ export default FeedsPostDetail;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->